sample/build.gradle

```diff
@@ -31,11 +31,11 @@ android {
             shrinkResources true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
-        debug {
-            minifyEnabled true
-            shrinkResources true
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
+//        debug {
+//            minifyEnabled false
+//            shrinkResources false
+//            proguardFiles getDefaultProguardFile('proguard-android-optim    ize.txt'), 'proguard-rules.pro'
+//        }
```

Comment:
we should have this on by default on the sample app. ok to disable it during local dev. but proguard is too easy to break, we need to test with it on.